config/locales/en.yml

```diff
@@ -543,6 +543,7 @@ en:
       people_watching_tag: "%{count} people are watching this tag"
       no_contributors_listed: "No contributors are listed."
       write_research_note: "Write a research note"
+      view_all: "View all"
```

Comment:
@RishiKumarRay I made a mistake here. Can you move line 546 below line 533. Thanks and sorry for the inconvenience.

Reply:
@TildaDares Done with these changes too, You can review them. Thanks